regions.go

```diff
@@ -22,6 +22,7 @@ const (
 	CapabilityVlans                  string = "Vlans"
 	CapabilityVPCs                   string = "VPCs"
 	CapabilityVPCsExtra              string = "VPCs Extra"
+	CapabilityVPU                    string = "NETINT Quadra T1U"
```

Comment:
Should we consider naming it as `CapabilityQuadraT1UVPU` just in case there will be more VPU models in the future?